### PR TITLE
Solid transparency (experimental)

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

File: .github/workflows/winci.yml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+          mkdir ${{ env.VCPKG_DEFAULT_BINARY_CACHE }}
+
+      - name: Restore vcpkg and its artifacts
+        uses: pat-s/always-upload-cache@v2.1.3
+        with:
+          path: |
+            ${{ env.VCPKG_DEFAULT_BINARY_CACHE }}
+            ${{ env.VCPKG_ROOT }}/installed
+          key: |
+            ${{ hashFiles( 'vcpkg.json' ) }}-${{ hashFiles( '.git/modules/vcpkg/HEAD' )}}-${{ env.TRIPLET }}
```
Comment (m-kuhn) [inline .github/workflows/winci.yml:64]:
```suggestion
            ${{ hashFiles( 'vcpkg.json' ) }}-${{ hashFiles( '.git/modules/vcpkg/HEAD' )}}-${{ env.TRIPLET }}-qgis
```

**Comment by m-kuhn** [comment]
Unrelated test failure

File: cmake/FindQGIS.cmake [hunk 2]
```diff
@@ -63,15 +70,15 @@ IF(WIN32)
...
-    # FIND_LIBRARY(QGIS_ANALYSIS_LIBRARY
-    #   NAMES qgis_analysis
-    #   PATHS
-    #     "$ENV{LIB_DIR}"
-    #     "$ENV{LIB}"
-    #     "$ENV{OSGEO4W_ROOT}/lib"
-    #     "$ENV{OSGEO4W_ROOT}/apps/${OSGEO4W_QGIS_SUBDIR}/lib"
-    #     "$ENV{PROGRAMFILES}/QGIS/lib"
-    # )
+    FIND_LIBRARY(QGIS_ANALYSIS_LIBRARY
```
Comment (nirvn) [inline cmake/FindQGIS.cmake:73]: Is that why the :apple: fails now (I remember seeing @3nids commenting this out during his iOS integration)? Should it be addressed in this PR or separately?

File: src/core/peliasgeocoder.h [hunk 3]
```diff
@@ -30,7 +31,7 @@
  *
  * \since QField 1.9
 */
-class CORE_EXPORT PeliasGeocoder : public QgsGeocoderInterface
+class QFIELD_CORE_EXPORT PeliasGeocoder : public QgsGeocoderInterface
```
Comment (nirvn) [inline src/core/peliasgeocoder.h:34]: Thanks for fixing my stuff sir :wink:

**Review by nirvn — APPROVED** [review]
Looking good -- just one question regarding iOS failure.

**Comment by nirvn** [comment]
Oh and also, is there a reason why we aren't getting APKs through comments in this PR?

**Comment by m-kuhn** [comment]
It's from an external repo, no secrets available

**Pull request closed without merging**